ArdusatSDK.cpp

```diff
@@ -90,7 +105,7 @@ const char * unit_to_str(uint8_t unit)
  *
  * @param sensorName name of sensor that failed.
  */
-void _beginError(prog_char *sensorName)
+void _beginError(prog_char sensorName[])
```

Comment:
I prefer the `char *var_name` style declaration to `char var_name[]` - they're equivalent, as far as the compiler is concerned, but the former is far more common.